.gitpod.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full
+
+USER gitpod
+
+RUN brew install shellcheck
```

Comment:
I would use shellcheck from distro instead of package manager designed for MacOS which does not seem to be concerned with security for linux by design?

Reply:
This is far easier to read and can be run as a non-root user

Reply:
Seems same to me in terms of readability

```dockerfile
RUN brew install shellcheck
```

```dockerfile
RUN apt install -y shellcheck
```

> and can be run as a non-root user

We can run apt as non-root as well without sacrificing security, but gitpod would require a contribution to be able to use it.

Alternatively we don't need non-root for the installation itself.

Reply:
Sure

Reply:
just remember to do apt-get update && apt install

Reply:
Done